src/app/timer/timer.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="timer">
+    <div class="timer__row">
```

Comment:
angular has incapsulated styles, so BEM is rarely used. You can continue using it if you like, but we also can use simpler class names